src/app/components/client/Chart.tsx

```diff
@@ -190,7 +191,7 @@ export const DoughnutChart = (props: Props) => {
                 ))}
               </tbody>
             </table>
-            {!props.hasRunScan ? prompt : null}
+            {!props.hasRunScan && props.isEligibleForFreeScan ? prompt : null}
```

Comment:
Nit: This could be just:
```suggestion
            {(!props.hasRunScan && props.isEligibleForFreeScan) && prompt}
```